ganga/GangaDirac/Lib/Server/DiracProcessManager.py

```diff
@@ -91,18 +92,26 @@ async def execute(self, cmd, args_dict=None, return_raw_dict=False, env=None, cr
         if not self.is_dirac_process_active(env_hash):
             self.start_dirac_process(dirac_env)
 
-        try:
-            task_id = uuid.uuid4()
-            task_done = self.manager.AioEvent()
-            await self.task_queues[env_hash].coro_put((task_done, task_id, cmd, args_dict))
-
-            await task_done.coro_wait()
-            dirac_result = self.task_result_dicts[env_hash].get(task_id)
-            del self.task_result_dicts[env_hash][task_id]
-
-            returnable = self.parse_command_result(dirac_result, str(cmd), return_raw_dict)
-            logger.debug(f'Executed DIRAC command {cmd} with result {returnable}')
-            return returnable
-        except RuntimeError:
-            logger.error('Attempted to add asyncio task after Ganga shutdown. Returning...')
-            return {'OK': False, 'Message': 'The event loop has shut down'}
+        # try:
+        task_id = uuid.uuid4()
+        task_done = self.manager.AioEvent()
+        await self.task_queues[env_hash].coro_put((task_done, task_id, cmd, args_dict))
+
+        await task_done.coro_wait()
+        dirac_result = self.task_result_dicts[env_hash].get(task_id)
+        del self.task_result_dicts[env_hash][task_id]
+
+        returnable = self.parse_command_result(dirac_result, str(cmd), return_raw_dict)
+        print(f'Executed DIRAC command {cmd} with result {returnable}')
```

Comment:
Any reason for the direct print statement here rather than a logger?

Reply:
Looks like I didn't push the latest changes, hence the commented out code, will fix it, thanks for catching that!